front-end/src/views/function/index.vue

```diff
@@ -1,27 +1,38 @@
 <template>
   <PageHeaderWrapper>
     <template #extra>
-      <el-button icon="el-icon-circle-plus-outline" type="primary" class="mr-4">
-        {{ $t('func.addFunc') }}
+      <el-button icon="el-icon-circle-plus-outline" type="primary" class="mr-4" @click="showAddFunc">
+        {{ $t("func.addFunc") }}
       </el-button>
     </template>
-    <Func v-loading="loading" :data="functionList" />
+    <Func :data="functionList" v-loading="loading" />
+    <add-func v-model="visibleAdd" :functionList="functionList" :refresh="refresh"/>
```

Comment:
Codacy has a fix for the issue: [Insert `·`](https://app.codacy.com/gh/FunctionStream/function-stream/pullRequest?prid=7834759)
```suggestion
    <add-func v-model="visibleAdd" :functionList="functionList" :refresh="refresh" />
```